lm_eval/tasks/flores_101.py

```diff
@@ -63,6 +68,184 @@ def max_generation_length(self):
         return 512
 
 
+class Flores101MT_fewshot_wmt_fr2en(Flores101MT):
```

Comment:
Add a 1 sentence class docstring describing diffs from the vanilla `Flores101MT` class that it's derived from.